app/assets/stylesheets/application.scss

```diff
@@ -2,6 +2,7 @@
 @import "theme/variables";
 @import "bootstrap";
 @import "theme/bootswatch";
+@import "theme/autocomplete";
```

Comment:
Would it actually be better to keep this to the `/components` folder? I would imagine we should keep code in `/theme` that we can easily overwrite with a vendor theme of sorts (something from [Bootswatch](https://bootswatch.com/)), and since we have custom code that wouldn't be doable here.

Also, if we have the CSS /JS / Markup all with the name of `autocomplete` it'll be easier to identify as a cohesive component.

What do you think?